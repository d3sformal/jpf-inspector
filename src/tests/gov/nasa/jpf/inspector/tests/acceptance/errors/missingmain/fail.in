hello
commence
commence
commence
commence
hello