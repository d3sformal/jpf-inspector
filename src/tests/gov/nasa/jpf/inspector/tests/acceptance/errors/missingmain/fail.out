Hello to you, too, dear user! The JPF Inspector command loop is working!
INFO: JPF created and connected, SuT is started
INFO: JPF Terminating
INFO: JPF created and connected, SuT is started
INFO: JPF Terminating
INFO: JPF created and connected, SuT is started
Hello to you, too, dear user! The JPF Inspector command loop is working!
INFO: JPF Terminating